internal/sourcemap/fetcher.go

```diff
@@ -38,6 +38,8 @@ type Fetcher interface {
 	//
 	// If there is no such source map available, Fetch returns a nil Consumer.
 	Fetch(ctx context.Context, name string, version string, bundleFilepath string) (*sourcemap.Consumer, error)
+
+	Wait()
```

Comment:
Please don't add a Wait method to the interface. This is irrelevant to all but one implementation, which is a good indicator that the method belongs only on the one implementation.

Reply:
Updated :+1: